src/landscape.cpp

```diff
@@ -1081,7 +1081,6 @@ static uint River_Hash(uint tile, uint dir)
 static void BuildRiver(TileIndex begin, TileIndex end)
 {
 	AyStar finder;
-	MemSetT(&finder, 0);
```

Comment:
What about the uninitialized members of AyStar?

Reply:
Not sure it matters too much - looks pretty similar to the only other usage of it in npf.cpp, but I've added a `= {0}` anyway